homework/index.js

```diff
@@ -30,13 +30,93 @@
     return elem;
   }
 
+  function renderRepository(leftContainer, repository) {
+    const tabel = createAndAppend('tabel', leftContainer, { id: 'tabel' });
+    const tr1 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr1, { class: 'col1_data', text: 'Repository:' });
+    createAndAppend('td', tr1, { class: 'col2_data', text: repository.name });
```

Comment:
The renderRepository function does its job well, but you can see there's a repeating structure here. You could improve this by: 
- extracting the repeating lines (from lines 35 to 38) into a separate function that renders a row, let's call it renderRow
- looping through an array of rows you want to render and calling renderRow, like this:
```
const rows = [
  { title: 'Repository', content: repository.name },
  { title: 'Description', content: repository.description },
  { title: 'Forks', content: repository.forks },
  { title: 'Updated', content: repository.updated_at }
];

rows.forEach((row) => {
  renderTableRow(tabel, row.title, row.content);
})
```
Also, I would suggest to use a different html structure than a table to be able to make this responsive. Maybe just some divs would work, which you could arrange in a layout using flex: https://css-tricks.com/snippets/css/a-guide-to-flexbox/.